STARTERKIT/webpack.config.js

```diff
@@ -0,0 +1,46 @@
+var webpack = require('webpack');
+
+var config = {
+  context: __dirname + '/js',
+  entry: {
+    app: './init.js',
+  },
+  output: {
+    path: __dirname + '/dist',
+    filename: 'app.js',
```

Comment:
How we will include this file to project theme?
Lets add reference to this file in libraries file  below https://github.com/skilld-labs/zen/blob/8.x-7.x/STARTERKIT/STARTERKIT.libraries.yml#L15